server.go

```diff
@@ -262,6 +276,14 @@ func (s *Server) processUnaryRPC(t transport.ServerTransport, stream *transport.
 			}
 		}()
 	}
+	monitor := s.opts.serverMonitor.NewServerMonitor(monitoring.Unary, stream.Method())
+	defer func() {
```

Comment:
No, I haven't. But the gRPC codebase seems to be full of defers. Should I put a `if monitor.(type) == monitoring.NoOpMonitor`?